packages/webdriverio/src/node/utils.ts

```diff
@@ -17,7 +17,5 @@ export async function assertDirectoryExists(filepath: string) {
  * @returns absolute file path
  */
 export function getAbsoluteFilepath(filepath: string) {
```

Comment:
Kept the function as there are some tests that spy on it, but if you'd like I can try and use `path.resolve` directly and see if the spy works correctly.